base/ltsockets.dtx

```diff
@@ -827,24 +827,24 @@
 \cs_new_protected:Npn \socket_new_plug:nnn #1#2#3 {
   \str_if_exist:cTF { l_@@_#1_plug_str }
       {
```

Comment:
One (remaining) case I didn't change: all occurrences of
```tex
  \str_if_exist:cTF { l_@@_#1_plug_str }
      {
      }
      {
      }
```
use 4-space indent, instead of 2.